packages/yourloops/test/integration/pages/home/home.hcp.spec.tsx

```diff
@@ -108,17 +108,6 @@ describe('HCP home page', () => {
     await renderHomePage()
 
     await testPatientManagementPrivateTeam()
-  })
-
-  it('should be able to manage the patient list when scoped on the private practice team', async () => {
```

Comment:
:raised_eyebrow: 

Why?

Reply:
I think the test already exists, doesn't it?

Reply:
Can you confirm that?